sdk/file_sdk/UnitTests/xmp_ut/xmp_file_tests.cpp

```diff
@@ -14,28 +21,89 @@ class Xmp_Tests : public QObject
   Xmp_Tests();
 
 private Q_SLOTS:
-  void GetTags_FileWithTag_ReturnCorrectTag();
+  void GetTags_FileWithAutomaticTag_ReturnCorrectTag();
+  void GetTags_FileWithManualTag_ReturnCorrectTag();
 };
 
-Xmp_Tests::Xmp_Tests()
-{
+Xmp_Tests::Xmp_Tests() {
 }
 
-void Xmp_Tests::GetTags_FileWithTag_ReturnCorrectTag()
-{
-  /*mip::Tag general( "f42aa342-8706-4288-bd11-ebb85995028c", "General", "", "vakuras@microsoft.com", true, "", "", mip::Method::MANUAL,"72f988bf-86f1-41af-91ab-2d7cd011db47" );
+bool VerifyTags(Tag tag1, Tag tag2) {

```

Comment:
>VerifyTags [](start = 5, length = 10)

can be useful for other tests

Reply:
made it static

---
In reply to: [136714723](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r136714723) [](ancestors = 136714723)